source/draft_api.rst

```diff
@@ -297,3 +297,41 @@ If the object does not exist, the API will return an Object Not Found error
 
 In case of any other error, the API will return a Server Error (see error
 docs).
+
+-----------------------
+
+Authorization and Roles
+-----------------------
+
+Each timesync user can be of one of two roles: user, and admin. In addition, each user
+has a role within each project to which they belong: member, data viewer, and project
+manager. These roles exist independently, and are defined by their permissions: a user
```

Comment:
this seems like a good place for listification: https://github.com/osuosl/docs/pull/138/files#diff-4036c50de2e6cbdbdb43d66bacb718ffR10